tests/graphql/transaction.spec.ts

```diff
@@ -75,6 +81,29 @@ describe("Transaction", () => {
       ]);
     });
 
+    it("can fetch filtered transactions", async () => {
+      stub.onFirstCall().resolves(
+        generatePaginatedResponse({
+          key: "transactions",
+          items: [firstTransaction, thirdTransaction],
+          pageInfo: { hasPreviousPage: false, hasNextPage: false },
+        }),
+      );
+
+      const results = await client.models.transaction.fetch({
+        filter: {
+          operator: BaseOperator.And,
+          amount_gt: 1000,
+          name_like:"SaNtA"
+        }
+      });
+
+      expect(results.items).to.deep.equal([
+        firstTransaction,
+        thirdTransaction,
+      ]);
```

Comment:
Can we extend the expectation a bit? E.g.
```
      sinon.assert.calledWithExactly(stub, sinon.match.string, {
        filter: {
          operator: BaseOperator.And,
          amount_gt: 1001,
          name_like: "SaNtA"
        }
      });
```

Without it the test runs fine even without the filter.

Reply:
Um good point 😂 test was pretty useless. What about this assertion 9aec03fb1bc6c0f27a521c4f75815ea354e38e1a ?

Reply:
👍 thank you